lib/Cake/Model/Datasource/DboSource.php

```diff
@@ -3459,12 +3469,16 @@ public function buildColumn($column) {
 			return null;
 		}
 
-		if (!isset($this->columns[$type])) {
-			trigger_error(__d('cake_dev', 'Column type %s does not exist', $type), E_USER_WARNING);
-			return null;
+		// if a storage requirement in bytes was set, try it first (i.e. 'integer/4' before 'integer')
+		if (isset($storage) && isset($this->columns[$type . '/' . $storage])) {
+			$real = $this->columns[$type . '/' . $storage];
```

Comment:
OK. So no need for a `'storage'` field anymore, you would rather have the storage requirements be conveyed by 2 new abstract types, correct?